src/languages/protobuf.js

```diff
@@ -6,36 +6,83 @@ Category: protocols
 */
 
 function(hljs) {
+  var KEYWORDS = {
+    keyword:
+      'syntax package import option optional required repeated group' +
+      'oneof returns',
+    built_in:
+      'double float int32 int64 uint32 uint64 sint32 sint64 ' +
+      'fixed32 fixed64 sfixed32 sfixed64 bool string bytes',
+    literal:
+      'true false'
+  };
+
   return {
-    keywords: {
-      keyword: 'package import option optional required repeated group oneof',
-      built_in: 'double float int32 int64 uint32 uint64 sint32 sint64 ' +
-        'fixed32 fixed64 sfixed32 sfixed64 bool string bytes',
-      literal: 'true false'
-    },
+    keywords: KEYWORDS,
     contains: [
+      // Quotes strings should be highlighted.
       hljs.QUOTE_STRING_MODE,
+
+      // Raw numbers should be highlighted also.
       hljs.NUMBER_MODE,
+
+      // Protobuf supports both // and /* */ style comments.
       hljs.C_LINE_COMMENT_MODE,
+      hljs.C_BLOCK_COMMENT_MODE,
+
+      // Messages, Services, and Enums
       {
         className: 'class',
-        beginKeywords: 'message enum service', end: /\{/,
-        illegal: /\n/,
```

Comment:
Was this supposed to be removed?  Might be why you're failing the auto-detect now.